test/visual/ButtonVisual.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import ButtonToolbar from 'react-bootstrap/lib/ButtonToolbar';
 import Button from 'react-bootstrap/lib/Button';
-import { Link } from 'react-router';
+import { Link } from 'react-router-dom';
 
 import LinkContainer from '../../src/LinkContainer';
 
 export default () => (
   <div>
-    <Link to="home">Back to Index</Link>
+    <Link to="/home">Back to Index</Link>
```

Comment:
Is this change something that you need to document in a changelog so that people know that they need to make it to their own code?